setup.py

```diff
@@ -622,6 +633,16 @@ def build_mx_extension(build_ext, options):
     build_ext.build_extension(mxnet_mpi_lib)
 
 
+def is_mx_cuda():
+    from mxnet import runtime
+    features = runtime.libinfo_features()
```

Comment:
This API is nice. Can we have an alternative approach to detect the cuda if the API is not there? In this way we do not lose Horovod support in the upcoming MXNet 1.4.0 release. Have you tried to detect the feature from libmxnet.so dependencies?